docs/dependency_confusion.md

```diff
@@ -61,80 +43,64 @@ to verify that the content of all direct and indirect dependencies is what you e
-> it.*
->
-> Npm packages follow the typical package file + lock file approach. The package file is
-> [package.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-json), the lock file is
-> [package-lock.json](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json) or
-> [npm-shrinkwrap.json](https://docs.npmjs.com/cli/v6/configuring-npm/shrinkwrap-json). Cachito
-> requires the lock file.
->
-> The lock file pins all dependencies to exact versions. For https dependencies, which are impossible
-> to pin, Cachito requires the integrity value (a checksum). For dependencies from the npm registry,
-> Cachito does not require integrity values, but newer versions of npm will always include them. Check
-> if all your dependencies have an integrity value, update your lock file if not.
->
-> Do not try to use private registries. If you point npm to a private registry (or any registry other
-> than the official one) via [.npmrc](https://docs.npmjs.com/cli/v6/configuring-npm/npmrc), Cachito
-> will ignore it and look in the official registry anyway.
->
-> ## yarn
->
-> *TL;DR: same as npm, but the handling of unofficial registries is slightly less surprising. Still,
-> you probably should not use them.*
->
-> Yarn packages are identical to npm packages except that they use
-> [yarn.lock](https://classic.yarnpkg.com/en/docs/yarn-lock/) as the lock file. Everything that
-> applies to npm applies to yarn.
->
-> The one difference is the handling of unofficial registries. If you point yarn to an unofficial
-> registry via .npmrc or [.yarnrc](https://classic.yarnpkg.com/en/docs/yarnrc), this will be reflected
-> in the resolved url in the lock file. Cachito will see that the url does not point to the official
-> registry and will treat the dependency as a plain https dependency. If the url is accessible to
-> Cachito, it will download the dependency directly without relying on npm/yarn dependency resolution.
-> That does not necessarily make using unofficial registries a good idea. If the registry is private,
-> your build will either fail or leak internal package names.
->
-> ## RubyGems (Bundler)
->
-> *TL;DR: allowing `https://rubygems.org` as the only source should mitigate the issue for GEM dependencies*
->
-> Cachito parses `Gemfile.lock` which pins all dependencies to exact versions. Cachito allows GEM dependencies
-> to be fetched only from `https://rubygems.org`, otherwise raises an error. GIT dependencies are specified using
-> a repo URL and pinned to a commit hash.
->
-> Bundler doesn't verify checksums of dependencies yet, however, there's an effort to bring
-> [internal GitLab implementation](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/92633)
-> to [the upstream](https://github.com/rubygems/rubygems/pull/5808).
-
-[cachito-1]: https://github.com/containerbuildsystem/cachito
+## pip
+
+*TL;DR: if your package is Cachito compliant, it is most likely safe. If you want to be sure, verify
```

Comment:
So you left the package managers paragraphs in, but we still mention cachito here, that's confusing.

Reply:
I am still figuring out what to put there because I don't want to include general statements that should be part of the regular documentation. Anyway, I dropped those paragraphs so it is less confusing.

Reply:
Note you can't just remove content for your convenience if it doesn't fit the immediate intention like in this PR, because we'd lose legitimate content too (and we'd have to revert the change in the future). IOW part of dropping references to the original cachito code base is also finding new home to the information that still applies instead of just outright deleting it, leaving it for a potential future follow up, that's not how things should be done. IMO it's better to have more-or-less updated content with some inaccuracies which anyone willing to do so can identify and fix than no content at all and so I didn't go too deep into our code to verify every single sentence in those paragraphs, but as I outlined in my comments, much of the information is still valid and so I'd imagine these bits to be moved to the respective package manager docs to a section clearly denoting the dependency confusion matter.

Reply:
I turned it back to its original state. It's not a high priority now, so I'll leave it like that.

I still don't know what should be in the package manager section; my intention is to avoid general notes that are part of the regular documentation. So, later on, we don't have to adjust docs at multiple places.